docker-compose.yml

```diff
@@ -219,6 +219,7 @@ services:
       image: nginx:mainline-alpine
       command: /bin/sh -c "envsubst < /etc/nginx/conf.d/templates/listen_plain.template > /etc/nginx/conf.d/listen_plain.active &&
         envsubst < /etc/nginx/conf.d/templates/listen_ssl.template > /etc/nginx/conf.d/listen_ssl.active &&
+        export ADD_WILDCARDS=$$(IFS=','; for S in $$ADDITIONAL_SUBDOMAINS; do A=$$A' '$$S'.*'; done; echo $$A) &&
```

Comment:
This works in all test cases?

Reply:
I find this ugly too, but did not find any way around it, to keep it as simple as it is right now.

The double dollars are for escaping docker-compose.
/bin/sh is from alpine, so should always give same behaviour.

If ADDITIONAL_SUBDOMAINS contains semicolons or is otherwise malformed, then it breaks the nginx config.
If its empty or contains a comma separated list, it works as expected.